src/components/table/partial-styles/tabulator-paginator.scss

```diff
@@ -49,6 +51,18 @@
     }
 }
 
+.has-no-pagination {
+    .tabulator-tableHolder {
+        height: 100% !important;
+        max-height: 100% !important;
+    }
+    .tabulator-footer {
+        visibility: hidden; //Display:none does not work here because of the tabulators dynamic calculations, which requires manual resizing.
+        opacity: 0;
+        transform: translateY(pxToRem(40));
+    }
```

Comment:
We can not remove the `tabulator-footer` class because for certain limetypes for example `deal` we have to show the `calculation holder` (I am not sure of the name, see the pic for clarification ). The class `.tabulator-calcs-holder` in side `.tabulator-footer` should not be hidden

<img width="879" alt="Screenshot 2020-11-12 at 10 16 44" src="https://user-images.githubusercontent.com/22968497/98921460-9f0b1300-24d1-11eb-959e-4186039f80d6.png">

Reply:
@maltja we can instead hide the pagination div (inside the footer), instead of hiding the entire footer div. However, the problem again is that Tabulator calculates the height of the table and inlines it differently when there is **a)** a footer only with pagination **b)** a footer with both pagination and calculations row

To fix this with CSS, we need to know when the footer contains calculations and add an extra class to the tabulator container, like what you are doing with `has-no-pagination`. You could add something like `has-calculations`.

Then we can address it with CSS

Reply:
Also, I prefer positive naming more :) instead of `has-no-xyz` I'd prefer `has-xyz` :) But it's a minor thing.

Reply:
Yes I also think we can solve it like that, I will take a look at it after the lime-project env meeting 🙂